.circleci/config.yml

```diff
@@ -263,9 +266,12 @@ jobs:
       - revenuecat/setup-git-credentials
       - setup-flutter
       - run:
-          name: Prepare next version
-          command: bundle exec fastlane update_hybrid_common version:<< pipeline.parameters.hybrid-common-version >> open_pr:true automatic_release:true
-
+          name: Updating to next version of phc
+          command: |
+            bundle exec fastlane update_hybrid_common 
+            version:<< pipeline.parameters.version >> \
```

Comment:
Hmm I see you are adding the `\` from the second line but not on the first one. Does it need it there as well?